run_dir/static/js/pricing_quote.js

```diff
@@ -95,18 +112,19 @@ app.component('v-pricing-quote', {
         },
         generate_quote:  function (event) {
           product_list = {}
+          var no_prod_error = 'Please add Products to generate a quote!';
           if(Object.keys(this.$root.quote_prod_ids).length > 0){
+            if(this.$root.any_errors)
+              this.$root.error_messages.pop()
```

Comment:
Didn't know you could skip the curly brackets... not sure I'm convinced it's a good thing for readability though?

Reply:
You can if its just a single line. I can add it if you want